/home/micha/.local/lib/python3.10/site-packages/matplotlib/projections/__init__.py:63: UserWarning: Unable to import Axes3D. This may be due to multiple versions of Matplotlib being installed (e.g. as a system package and as a pip package). As a result, the 3D projection is not available.
  warnings.warn("Unable to import Axes3D. This may be due to multiple versions of "
Warning: Teleporting vehicle '7'; collision with vehicle '6', lane='1/1to2/1_0', gap=-0.73, time=5.00, stage=move.
Warning: Vehicle '7' ends teleporting on edge '2/1to3/1', time=5.00.
Warning: Teleporting vehicle '20'; collision with vehicle '19', lane='1/1to2/1_0', gap=-0.26, time=34.00, stage=move.
Warning: Vehicle '20' ends teleporting on edge '2/1to3/1', time=34.00.
Warning: Teleporting vehicle '18'; collision with vehicle '17', lane='1/1to2/1_0', gap=-0.77, time=52.00, stage=move.
Warning: Teleporting vehicle '24'; collision with vehicle '23', lane='1/1to2/1_0', gap=-0.15, time=52.00, stage=move.
Warning: Vehicle '18' ends teleporting on edge '2/1to3/1', time=52.00.
Warning: Vehicle '24' ends teleporting on edge '2/1to3/1', time=52.00.
